pdm.lock

```diff
@@ -4,8 +4,11 @@
 [metadata]
```

Comment:
was it generated on MacOS? seems a bit weird that so many changes were incorporated

Reply:
I used a fedora linux machine.